### PR TITLE
kuguk - add google search test

### PR DESCRIPTION
**Pull request by kuguk**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-chrome-driver</artifactId>
             <scope>test</scope>
-            <version>${selenium.java.version}</version>       
+            <version>${chrome.driver.version}</version>
```
Comment (toleksyn) [inline pom.xml]: revert this, use ${selenium.java.version}

File: src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java [hunk 2]
```diff
@@ -6,25 +6,27 @@
...
 		searchResultsLinks = new GoogleHomePage()
 				.open()
-				.searchFor(searchString)
+				.searchFor("funny kitten")
 				.getSearchResultsLinks();
-	}
-
+	assertTrue
+	        (searchResultsLinks.toString().toLowerCase().contains("funny") 
+	        && searchResultsLinks.toString().toLowerCase().contains("kitten")); }
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java]: combine into "funny kitten"

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (kuguk) [reply pom.xml]: fixed

File: src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java (hunk 2 above)
Comment (kuguk) [reply src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java]: fixed

File: src/main/java/com/softserveinc/ita/kuguk/GoogleSearchResultsPage.java [hunk 3]
```diff
@@ -4,16 +4,20 @@
...
+import java.util.stream.Stream;
 
 import com.softserveinc.ita.pageobjects_task.kuguk.TestRunner;
 
 public class GoogleSearchResultsPage {
 
 	public List<String> getSearchResultsLinks() {
-
+		return 	Stream.of (TestRunner.driver.findElements(By.xpath("//h3[@class='LC20lb DKV0Md']")))  		
+				.map(foundLinksList -> foundLinksList.get(0).getText ())
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/kuguk/GoogleSearchResultsPage.java]: you're mapping each element onto first link text

File: src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java [hunk 4]
```diff
@@ -6,25 +6,26 @@
...
 		searchResultsLinks = new GoogleHomePage()
 				.open()
-				.searchFor(searchString)
+				.searchFor("funny kitten")
 				.getSearchResultsLinks();
-	}
-
+	assertTrue
+	        (searchResultsLinks.toString().toLowerCase().contains("funny kitten")); }
+
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java]: do not use custom formatting
assertTrue();

code format before commiting

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/kuguk/GoogleSearchResultsPage.java (hunk 3 above)
Comment (kuguk) [reply src/main/java/com/softserveinc/ita/kuguk/GoogleSearchResultsPage.java]: fixed

File: src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java (hunk 4 above)
Comment (kuguk) [reply src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java]: fixed

File: src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java [hunk 5]
```diff
@@ -6,25 +6,27 @@
...
 
 import static org.testng.Assert.assertTrue;
-
 import java.util.List;
 
 public class GoogleTest extends TestRunner {
 
 	private List<String> searchResultsLinks;
-	private String searchString = "smartphone";
+	private GoogleHomePage openGoogleHome;
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/pageobjects_task/kuguk/GoogleTest.java:14]: rename

**Review by toleksyn — CHANGES_REQUESTED** [review]

**Pull request closed without merging**